apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,36 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.get
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ListAssertionSamples {
+
+    // subject under test
+    private val sut = listOf(1, 2, 3)
```

Comment:
Have a look at the rebiew for pairAssertions the same applies here

Reply:
ok